packages/fxa-settings/src/components/ConnectedServices/index.test.tsx

```diff
@@ -25,6 +33,58 @@ const getIconAndServiceLink = async (name: string, testId: string) => {
   };
 };
 
+const firstSyncMock = DESKTOP_SYNC_MOCKS[0];
+const mocks = [
+  {
+    request: {
+      query: ATTACHED_CLIENT_DISCONNECT_MUTATION,
+      variables: {
+        input: {
+          clientId: firstSyncMock.clientId,
+          deviceId: firstSyncMock.deviceId,
+          refreshTokenId: firstSyncMock.refreshTokenId,
+          sessionTokenId: firstSyncMock.sessionTokenId,
+        },
+      },
+    },
+    result: {
+      data: {},
+    },
+  },
+];
+
+const clickFirstSignOutButton = async () => {
+  await act(async () => {
+    const signOutButtons = await screen.findAllByTestId(
+      'connected-service-sign-out'
+    );
+    fireEvent.click(signOutButtons[0]);
+  });
+  await wait();
```

Comment:
What does this do when you don't wait for anything in particular?

Reply:
Not sure? I assume a `setTimeout` to flush the UI loop, but I don't know. I was following the approach used in the other tests in this file, though this method is marked deprecated in favor of `waitFor`, which includes an explicit wait condition.